src/commands/create.js

```diff
@@ -50,7 +51,7 @@ class CreateCommand extends ApifyCommand {
         actorName = await ensureValidActorName(actorName);
         let messages = null;
         if (manifestPromise) {
-            ({ archiveUrl: templateArchiveUrl, skipOptionalDeps, messages } = await getTemplateDefinition(templateName, manifestPromise));
+            ({ archiveUrl: templateArchiveUrl, skipOptionalDeps, messages, localReadmeSuffix } = await getTemplateDefinition(templateName, manifestPromise));
```

Comment:
The `localReadmeSuffix` is in the manifest, not in the template definition, this won't work